README.md

```diff
@@ -73,6 +73,23 @@ MySQL search.
 kellerkinder.enableElasticsearch = true;
 ```
 
+### kellerkinder.documentRoot
+Changes the default document root (`public`) of caddy to the specified value
+
+*_Example for the current folder_*
+```
+kellerkinder.documentRoot = ".";
+```
+### kellerkinder.defaultMatcherPaths
```

Comment:
Maybe more along the lines of `staticFilePaths` to clarify what those are expected to be and hint at how caddy uses them?

Reply:
Good point - was adjusted